test/browser_tests/page_objects/page_dbwu-small-businesses.js

```diff
@@ -26,7 +26,7 @@ function SmallBusinessess() {
 
   this.smallBusinessLinks = this.smallBusinessInfo.all( by.css( 'a' ) );
 
-  this.awardsTable = element( by.css( '.content_main .simple-table' ) );
+  this.awardsTable = element( by.css( '.content_main .o-table' ) );
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2289/commits/1fe6a8f4a7d73e52617484e96e5b0df5b3e33756. There are two pages that contain tables that originate in Wordpress. We will need to switch those tables to the new `o-table` class. Although, I think they look better without the `simple-table` class on mobile.